docs/kb/install-knative.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: docs
+toc_id: install-knative
+title: VMware Event Broker Appliance - Knative
+description: Deploying VMware Event Broker Appliance with Knative
+permalink: /kb/install-knative
+cta:
+ title: Deploy a Function
+ description: At this point, you have successfully deployed the VMware Event Broker Appliance and you are ready to start deploying your functions! 
+ actions:
+  - text: Check the [pre-built Functions](/examples) to quickly get started
+  - text: Find the instruction to deploy a function - [here](use-functions).
+---
+# Deploy VMware Event Broker Appliance with Knative
+
+Customers looking to seamlessly extend their vCenter by either deploying our prebuilt functions or writing your own functions can get started quickly by deploying VMware Event Broker Appliance with Knative as the Event Processor
+
+## Appliance Deployment Steps
+
+### Requirements
+
+* 2 vCPU and 8GB of memory for VMware Event Broker Appliance
+* vCenter Server 6.x or greater
+* Account to login to vCenter Server (readOnly is sufficient)
+
+### Step 1 - Download the VMware Event Broker Appliance (OVA) from the [VMware Fling site](https://flings.vmware.com/vmware-event-broker-appliance){:target="_blank"}.
```

Comment:
Would rather put the descriptions on separate lines because it looks strange with longer text (especially step 3), e.g.

```suggestion
### Step 1

Download the VMware Event Broker Appliance (OVA) from the [VMware Fling site](https://flings.vmware.com/vmware-event-broker-appliance){:target="_blank"}.
```